### PR TITLE
Adds jujushell buttons for the unit in error state

### PR DESCRIPTION
**Pull request by jaycee**

### CODE REVIEW

**Comment by jaycee** [comment]
QA:

(Use the prod env for this)

* Induce an error state in a unit in a model you control (or ask for access to my model)
* Check that the debug-hooks and tail logs buttons are available for that unit, and that they do the expected functions
* Check that the buttons are not available for a unit not in error

File: jujugui/static/gui/src/app/components/inspector/unit-details/unit-details.js [hunk 1]
```diff
@@ -25,17 +25,26 @@ class UnitDetails extends React.Component {
         }}});
   }
 
+  /**
+    Open the terminal and run debug-hooks on the unit
+
+    @method _debugHooks
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/inspector/unit-details/unit-details.js]: `@method` is no longer required

File: jujugui/static/gui/src/app/components/inspector/unit-details/unit-details.js [hunk 2]
```diff
@@ -25,17 +25,26 @@ class UnitDetails extends React.Component {
...
+  _debugHooks() {
+    const unit = this.props.unit;
+    let commands = [`juju debug-hooks ${unit.id}`];
+    this.props.changeState({terminal: commands});
+  }
+
   /**
     Open the terminal and SSH to the unit.
 
     @method _sshToUnit
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/inspector/unit-details/unit-details.js]: see above

also missing `@param`

**Comment by makyo** [comment]
QA okay

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8716/

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8719/

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge/2833

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8720/

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge/2834

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**